src/main/java/pvpmode/internal/server/PvPServerEventHandler.java

```diff
@@ -42,12 +43,18 @@ public PvPServerEventHandler ()
     }
 
     /**
-     * Cancels combat events associated with PvP-disabled players. Note that this
-     * function will be invoked twice per attack - this is because of a Forge bug.
+     * Cancels combat events associated with PvP-disabled players. Note that
+     * this function will be invoked twice per attack - this is because of a
+     * Forge bug, but the {@link PvPCommonUtils isCurrentAttackDuplicate} call
```

Comment:
```suggestion
     * Forge bug, but the {@link PvPCommonUtils#isCurrentAttackDuplicate()} call
```
Possibly the parentheses after the method are not required, not too sure.

Reply:
It works with them, so they might as well be added.